src/components/ec-button-group/ec-button-group.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="ec-btn-group">
+    <button
+      v-for="(button, index) in items"
```

Comment:
I'd name the button as `item` or I'd rename items to `buttons` then (I strongly prefer the first approach)